changes/924.deprecation.md

```diff
@@ -0,0 +1,3 @@
+Deprecated `RESTClient.command_builder` and `RESTClient.create_application_command`.
+
```

Comment:
The clear menu methods have nothing to do with this deprecation

Reply:
What exactly do you mean?

Reply:
command_builder is a deprecated alias for slash_command_builder and create_application_command is a deprecated alias for create_slash_command; create_context_menu_command and context_menu_builder are unrelated new functionality